llvm/lib/Target/AMDGPU/GCNSubtarget.h

```diff
@@ -1837,7 +1839,7 @@ class GCNSubtarget final : public AMDGPUGenSubtargetInfo,
   }
 
   /// \returns true if the subtarget supports clusters of workgroups.
-  bool hasClusters() const { return GFX1250Insts; }
+  bool hasClusters() const { return HasCluster; }
```

Comment:
Please call the target feature "HasClusters". The inconsistency is really annoying.

Reply:
These should have generated names from tablegen, we just didn't define the macro for it and ended up reinventing all of these getters

Reply:
https://github.com/llvm/llvm-project/pull/162103